drivers/ethernet/nxp_enet/eth_nxp_enet.c

```diff
@@ -11,6 +11,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "zephyr/sys_clock.h"
```

Comment:
Unclear here- why are we adding the `sys_clock.h` include here? Was it always needed, and this PR just caught the requirement for the include?

Reply:
this is quite a mystery, my suspect lineup involves a set of various dubious mouse "controls" that tend to serve no purpose except annoy me